matrix/lambdas/api/v0/core.py

```diff
@@ -126,17 +126,19 @@ def get_matrix(request_id: str):
 
         matrix_location = ""
         if format == MatrixFormat.LOOM.value:
-            matrix_location = f"https://s3.amazonaws.com/{matrix_results_bucket}/{request_id}.{format}"
+            matrix_location = f"https://s3.amazonaws.com/{matrix_results_bucket}/" \
+                              f"{request_tracker.s3_results_prefix}/{request_id}.{format}"
         elif format == MatrixFormat.CSV.value or format == MatrixFormat.MTX.value:
-            matrix_location = f"https://s3.amazonaws.com/{matrix_results_bucket}/{request_id}.{format}.zip"
+            matrix_location = f"https://s3.amazonaws.com/{matrix_results_bucket}/" \
+                              f"{request_tracker.s3_results_prefix}/{request_id}.{format}.zip"
 
         return ({'request_id': request_id,
                  'status': MatrixRequestStatus.COMPLETE.value,
                  'matrix_location': matrix_location,
                  'eta': "",
                  'message': f"Request {request_id} has successfully completed. "
                             f"The resultant expression matrix is available for download at "
-                            f"{matrix_location}"},
+                            f"{matrix_location} for 30 days."},
```

Comment:
What happens if someone makes a request, waits 29 days, and then does a GET? I think they're get this 30 day message even though the object is about to be deleted.

Reply:
Good call. The expiration date (set on object creation as the creation date + bucket expiration policy lifetime) can be retrieved via [head_object](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/s3.html#S3.Client.head_object) and can provide a correct expiration.

However, am removing this change for now and will reintroduce when expiration is implemented.